lab1.js

```diff
@@ -76,14 +79,28 @@ var sentence2 = 'Come over here so you can scratch my belly.';
 
 // TODO: part #1: use a for loop to replace the words in sentence 1 with
 // 'chirp' (10 points)
-
-// your code goes here
+sentence1 = sentence1.split(' ');
+for (var i = 0; i < sentence1.length; i++) {
+  sentence1[i] = 'chirp';
+}
 
 // TODO: part #2: use a while or do-while loop to replace the words in sentence 2
 // with 'chirp' (10 points)
+sentence2 = sentence2.split(' ');
+var isDone = false;
+var loopIndex = 0;
+
+while (!isDone) {
```

Comment:
like the use of a "flag" (a boolean) here - this concept is something you can use in lots of other places.